ui/k-info-panel/switch_info.kytos

```diff
@@ -1,7 +1,8 @@
 <template>
     <k-accordion>
+      <button class='bt_state' @click= "bt_stateToggle">{{this.nextState}}</button>
```

Comment:
@Alopalao the toggle button ended up great, however I'd encourage you to still use a `k-button` here since `k-button` can also parametrize everything you need, for instance:

```
+      <k-button class='bt_state' :on_click="bt_stateToggle" :title="nextState"></k-button>
```

Reply:
Changed it, but I did not add `class='bt_state'` since the css part would not be needed .

Reply:
Changed it, but I did not add class='bt_state' since the css part would not be needed. Commit [4e91ab3ee60976a1d6fabfb1cb041793c253b9ed](https://github.com/kytos-ng/topology/pull/70/commits/4e91ab3ee60976a1d6fabfb1cb041793c253b9ed)